index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Template 1-mobile-main</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class="logo">
+        <a href="#">My logo</a>
+        <a><img src="./icons/menu.png" alt="menu"></a>
+    </nav>
+    <div class="home-container">
+            
+            <section class="main-info">
+                <div class="introduction">
+                       <h1>I’m Moriasi glad to see you!</h1>
```

Comment:
- [ ] You are almost there! Let's use the correct color for the main title as the previous reviewer said, this is the color provided in the figma template:

![Captura de pantalla 2022-05-27 a la(s) 3 44 37 p  m](https://user-images.githubusercontent.com/83717285/170787372-ef15ca68-26fe-491b-a87e-9a86a7672bbc.png)